### ISSUE TITLE
Wrong parameter name in readme.txt

### ISSUE THREAD

**Issue by UselessGuru** [develsoftware/GMinerRelease#91]
'--cuda_devices' - list devices available for mining

does no longer exist.

It should say '--list_devices'

**Comment by UselessGuru** [comment]
Issue is still present in 2.23

**Comment by mvandemar** [comment]
@UselessGuru Looks like it was fixed in 2.24:
```
$ miner --help
+----------------------------------------------------------------+
|                          GMiner v2.24                          |
+----------------------------------------------------------------+
Allowed options:
  -h [ --help ]               display this message
  -v [ --version ]            print program version
  --list_devices              display available GPUs
```

**Comment by develsoftware** [comment]
Fixed in 2.25
